benchmark_runner/main/environment_variables.py

```diff
@@ -31,6 +31,23 @@ def __init__(self):
         self._environment_variables_dict['elasticsearch'] = os.environ.get('ELASTICSEARCH', '')
         self._environment_variables_dict['elasticsearch_port'] = os.environ.get('ELASTICSEARCH_PORT', '')
 
+        # This parameter 'True'/'False' for debug workload, when its set to 'True' it will stop the workload when finishing
+        self._environment_variables_dict['stop_when_workload_finish'] = os.environ.get('STOP_WHEN_WORKLOAD_FINISH', '')
+
+        # default parameter - change only if needed
+        # Parameters below related to 'run_workload()'
+        self._environment_variables_dict['workloads'] = ['stressng_pod', 'stressng_vm', 'uperf_pod', 'uperf_vm', 'hammerdb_pod_mariadb', 'hammerdb_vm_mariadb',  'hammerdb_pod_postgres', 'hammerdb_vm_postgres', 'hammerdb_pod_mssql', 'hammerdb_vm_mssql']
```

Comment:
This is really a constant.

Reply:
This isn't something the user can meaningfully modify, right?  They would need to add code to do so.